### ISSUE TITLE
Pi has more digits than 6

### ISSUE THREAD

**Issue by fenrus75** [CGAL/cgal#4445]
https://github.com/CGAL/cgal/blob/76ff945bf47e9d6a123bf51670ba2f5fc88c32ea/Straight_skeleton_2/include/CGAL/Straight_skeleton_2/Straight_skeleton_builder_2_impl.h#L1424

CGAL overall has a few different definitions of PI that have better accuracy than this one...  I would suggest using one or just us M_PI from math.h

**Comment by lrineau** [comment]
There are actually several hard-coded version of pi:
```shell-session
% ack --cpp '\b3\.141' ~/Git/cgal/*/include
/home/lrineau/Git/cgal/Generator/include/CGAL/point_generators_d.h
59:    const double pi = 3.141592653589793238462643;
107:    const double pi = 3.141592653589793238462643;

/home/lrineau/Git/cgal/Number_types/include/CGAL/number_type_config.h
22:#define CGAL_PI 3.14159265358979323846

/home/lrineau/Git/cgal/Point_set_processing_3/include/CGAL/bilateral_smooth_point_set.h
146:  FT cos_sigma = cos(sharpness_angle / 180.0 * 3.1415926);

/home/lrineau/Git/cgal/Ridges_3/include/CGAL/Umbilics.h
233:  const double  pi=3.141592653589793;

/home/lrineau/Git/cgal/Snap_rounding_2/include/CGAL/Snap_rounding_kd_2.h
397:    pi(3.1415), half_pi(1.57075),

/home/lrineau/Git/cgal/Straight_skeleton_2/include/CGAL/Straight_skeleton_2/Straight_skeleton_builder_2_impl.h
1424:  double rad  = atan >= 0.0 ? atan : 2.0 * 3.141592 + atan ;
1425:  double deg  = rad * 180.0 / 3.141592 ;
```

They should all use `CGAL_PI` defined in `<CGAL/number_type_config.h>`.

**Comment by lrineau** [comment]
Fixed by #4450 in `master`.
